packages/workspace/src/affine/__tests__/api.spec.ts

```diff
@@ -208,4 +212,30 @@ describe('api', () => {
       timeout: 30000,
     }
   );
+
+  test(
+    'usage',
+    async () => {
+      const usageResponse = await userApis.getUsage();
+      usageResponseSchema.parse(usageResponse);
+      const id = await createWorkspace(workspaceApis);
+      const path = require.resolve('@affine-test/fixtures/smile.png');
+      const imageBuffer = await readFile(path);
+      const blobId = await workspaceApis.uploadBlob(
+        id,
+        imageBuffer,
+        'image/png'
+      );
+      const buffer = await workspaceApis.getBlob(blobId);
+      expect(buffer.byteLength).toEqual(imageBuffer.byteLength);
+      const newUsageResponse = await userApis.getUsage();
+      usageResponseSchema.parse(newUsageResponse);
+      expect(usageResponse.blob_usage.usage).not.equals(
```

Comment:
/cc @zuoxiaodong0815 

`getUsage` won't return the correct usage after we upload a blob